app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    commands = command.split()
+
+    if len(commands) != 3 or commands[0] != "mv":
+        raise ValueError(f"'{command}': invalid command input")
+
+    old_file_name, new_file_name = commands[1:]
+
+    old_file_tail = os.path.split(old_file_name)[1]
+    new_file_tail = os.path.split(new_file_name)[1]
+    if not old_file_tail or not new_file_tail:
+        raise ValueError(f"'{command}' must contain two file names")
+
+    new_file_path = os.path.split(new_file_name)[0]
```

Comment:
Check overall logic of your code - it fails when destination path is a directory:
`mv file.txt first_dir/second_dir/`

Reply:
It handled on lines 12 to 15
Code raises `ValueError` if there is no filenames in commands but only path.

Reply:
> If name in destination path ends with / it must be considered as a directory. 

There shouldn't be a ValueError in this case. if you execute `mv file.txt first_dir/second_dir/` it assumes that you will move file.txt inside first_dir/second_dir/